### PR TITLE
Adding resource quota metrics to rules

### PR DESCRIPTION
**Pull request by Anxhela21**
Resource quota metrics to be added to the grafana dashboard as per request. This PR has the changes needed to the prometheus recording rules to include those metrics.

### CODE REVIEW

File: data-assets/acm_right_sizing_grafana_dashboard.yaml [hunk 1]
```diff
@@ -31,7 +31,11 @@ data:
       "gnetId": null,
       "graphTooltip": 0,
       "id": 15,
+<<<<<<< HEAD
```
Comment (dvandra) [inline data-assets/acm_right_sizing_grafana_dashboard.yaml:34]: I think this conflict should be resolved.

File: data-assets/acm_right_sizing_grafana_dashboard.yaml [hunk 2]
```diff
@@ -173,7 +177,11 @@ data:
           "targets": [
             {
               "exemplar": true,
+<<<<<<< HEAD
```
Comment (dvandra) [inline data-assets/acm_right_sizing_grafana_dashboard.yaml:180]: Here also

File: data-assets/acm_right_sizing_grafana_dashboard.yaml [hunk 3]
```diff
@@ -1663,16 +1683,55 @@ data:
...
+        "from": "now-90d",
         "to": "now"
       },
       "timepicker": {},
       "timezone": "",
       "title": "ACM Right-Sizing Namespace",
       "uid": "bnqQIPySE",
+<<<<<<< HEAD
       "version": 2
+=======
```
Comment (dvandra) [inline data-assets/acm_right_sizing_grafana_dashboard.yaml:1734]: Here.

File: data-assets/acm_right_sizing_grafana_dashboard.yaml (hunk 1 above)
Comment (Anxhela21) [reply data-assets/acm_right_sizing_grafana_dashboard.yaml:34]: did not make these changes that commit came from main

File: data-assets/acm_right_sizing_grafana_dashboard.yaml (hunk 2 above)
Comment (Anxhela21) [reply data-assets/acm_right_sizing_grafana_dashboard.yaml:180]: same with here

File: data-assets/acm_right_sizing_grafana_dashboard.yaml (hunk 3 above)
Comment (Anxhela21) [reply data-assets/acm_right_sizing_grafana_dashboard.yaml:1734]: and here

File: data-assets/acm_right_sizing_grafana_dashboard.yaml (hunk 1 above)
Comment (Anxhela21) [reply data-assets/acm_right_sizing_grafana_dashboard.yaml:34]: done.

**Review by dvandra — APPROVED** [review]

**Pull request merged**